src/domain/occurrences/fields/OccurrenceTimeRangeField.tsx

```diff
@@ -28,8 +28,4 @@ const OccurrenceTimeRangeField = ({
   );
 };
 
-OccurrenceTimeRangeField.defaultProps = {
-  label: 'occurrences.fields.time.fields.time.label',
-};
-
```

Comment:
Does removing this affect functionality? If it does, should the functionality be added back in some other way? Or is the functionality not needed?